meerk40t/gui/wxmmain.py

```diff
@@ -1006,14 +1009,13 @@ def part_of_group():
         kernel.register(
             "button/align/AlignLeft",
             {
-                "label": _("Align Left"),
+                "label": _("Left"),
                 "icon": icons8_align_left_50,
                 "tip": _(
                     "Align selected elements at the leftmost position (right click: of the bed)"
                 ),
-                "action": lambda v: kernel.elements("align left\n"),
-                "right": lambda v: kernel.elements("align bedleft\n"),
-                "size": buttonsize,
+                "action": lambda v: kernel.elements("alignmode push\nalignmode first\nalign left\nalignmode pop\n"),
+                "right": lambda v: kernel.elements("alignmode push\nalignmode bed\nalign left 1\nalignmode pop\n"),                "size": buttonsize,
```

Comment:
An align mode language? I might need to check into this in more detail. Why isn't this something more like: `align bed left 1` or `align --bed left 1` or something like that? What about this requires a stack?